From b98726401806ce05a4af7f10d2248612621da878 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Mon, 6 Jun 2016 16:52:52 +0800
Subject: [PATCH 1083/5242] MLK-12887: ARM: dts: fix noise issue for mono
 playback

commit  f2ddc70dfc99638673bb1d6bff1504341ef7999e from
https://source.codeaurora.org/external/imx/linux-imx.git

Using pull-up or pull down will cause that codec can get
big data in right channel. using keeper to fix this issue.

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
index 67e4e87..3e68c6d 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
@@ -647,7 +647,7 @@
 				MX6UL_PAD_SD1_DATA0__SAI2_TX_SYNC     0x1b0b0
 				MX6UL_PAD_SD1_DATA1__SAI2_TX_BCLK     0x1b0b0
 				MX6UL_PAD_SD1_DATA2__SAI2_RX_DATA     0x110b0
-				MX6UL_PAD_SD1_DATA3__SAI2_TX_DATA     0x1f0b8
+				MX6UL_PAD_SD1_DATA3__SAI2_TX_DATA     0x110b0
 				MX6UL_PAD_SD1_CLK__SAI2_MCLK          0x1b0b0
 			>;
 		};
-- 
1.7.9.5

